gpytorch/kernels/kernel.py

```diff
@@ -318,6 +368,7 @@ def _covar_dist(
             x2 = x2.unsqueeze(0).transpose(0, -1)
```

Comment:
Can we make `covar_dist` a public method? It's really confusing right now to tell what is a helper method and what's not.